cmd/operator-sdk/bundle/validate.go

```diff
@@ -40,22 +40,20 @@ func newValidateCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "validate",
 		Short: "Validate an operator bundle image",
-		Long: `The 'operator-sdk bundle validate' command can validate both content and
-format of an operator bundle image or an operator bundles directory on-disk
-containing operator metadata and manifests. This command will exit with a non-zero
-exit code if any validation tests fail.
+		Long: `
```

Comment:
These are unnecessary cosmetic changes.

Reply:
I am not sure if that I can agree that it `unnecessary cosmetic` since it is what ensures that docs layout will not be broken as the command-line tool will show the information clearly.